packages/base-driver/lib/basedriver/core.ts

```diff
@@ -308,11 +316,15 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
 
     if (!_.includes(validStrategies, strategy)) {
       throw new errors.InvalidSelectorError(
-        `Locator Strategy '${strategy}' is not supported for this session`
+        `Locator Strategy '${strategy}' is not supported for this session`,
       );
     }
   }
 
+  getBidiProxy(): string | null {
```

Comment:
getBidiProxy sounds like we want to return an object. If we only need the proxy url then getBidiProxyUrl would probably match better